src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/SolutionPickerView.xaml

```diff
@@ -0,0 +1,48 @@
+<UserControl x:Class="NuGet.PackageManagement.UI.SolutionPickerView"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:nuget="clr-namespace:NuGet.PackageManagement.UI"
+             xmlns:local="clr-namespace:NuGet.PackageManagement.UI"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="800"
+             x:Name="_solPickerView">
+  <Grid>
+    <TextBlock Text="{x:Static nuget:Resources.Text_OpenInNuGetPackageManager}"
+               FontSize="24"
+               Margin="75,10,0,0"
+               FontWeight="Bold"/>
+    <TextBlock Text="{x:Static nuget:Resources.Text_RecentProjectsSolutions}"
+               FontSize="16"
+               Margin="75,75,0,0"
+               FontWeight="Bold"/>
+    <ListBox ItemsSource="{Binding SolutionList}"
+             Margin="65, 110"
+             BorderThickness="0"
+             HorizontalContentAlignment="Stretch">
+      <ListBox.ItemTemplate>
+        <DataTemplate>
+          <Button
+            Command="{Binding OpenSolutionCommand}"
+            CommandParameter="{Binding FullPath}"
+            IsCancel="True"
+            Background="Transparent"
+            BorderThickness="0"
+            HorizontalContentAlignment="Stretch"
+            Padding="{Binding RelativeSource={RelativeSource Mode=FindAncestor, AncestorType={x:Type ListBoxItem}}, Path=Padding}">
+            <TextBlock TextWrapping="Wrap">
+             <Run
+               Text="{Binding SolutionName, Mode=OneWay}"
+               FontSize="16"/>
+             <LineBreak/>
+             <LineBreak/>
+             <Run Text="{Binding FilePath, Mode=OneWay}"/>
+             <LineBreak/>
+            </TextBlock>
```

Comment:
It would be better to implement this with two textblocks with margins instead which would give more control over the spacing rather than using linebreaks and a run, etc. Use a StackPanel as the container for the textblocks.